exercises/go-counting/.meta/src/reference/java/GoCounting.java

```diff
@@ -0,0 +1,156 @@
+import java.awt.Point;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
+
+class GoCounting {
+	int NONE = 0;
+	int WHITE = 1;
+	int BLACK = 2;
+	
+	int[][] board;
+	
+	GoCounting (String boardString) {
+		String[] lines = boardString.split("\n");
+		
+		board = new int[lines[0].length()][lines.length];
+		
+		for(int i = 0; i < lines.length; i++) {
```

Comment:
Minor nitpick: there should be a space after each `for` and after each `if`, e.g.
```
for (int i = 0; i < lines.length; i++) {
```
🙂